content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -674,13 +676,12 @@ Acceptation
 ~~~~~~~~~~~
 
 If all the commercial information is correct on your vendor bill, you can accept the document using
-the button: Aceptar Documento, once this is done the DTE Acceptation Status  changes to: Accepted and
-an email of acceptance is sent to the vendor. 
+the button: Aceptar Documento, once this is done the DTE Acceptation Status  changes to: Accepted
```

Comment:
Fixed this.

To note, however, I did not search for `:guilabel:` opportunities outside the main body of content added in this PR given that the old content was created before `:guilabel:` was introduced. 

That being said, I'm _more_ than happy to revise content outside the additions made on this PR (since this doc could use a cleanup anyway 😅). If not this PR, we can have a new writer/hire clean up the doc for RST training. 

Please advise what you'd like to do for scope of this PR, thank you 🙏